model/minsky.cc

```diff
@@ -264,7 +264,12 @@ namespace minsky
     GroupPtr g(new Group);
     canvas.setItemFocus(model->addGroup(g));
```

Comment:
I don't think this line is needed at all, although you undo it later on on line 272.